library/maven/artifacts.bzl

```diff
@@ -234,30 +234,32 @@ artifacts = {
     "org.hamcrest:hamcrest-core": "1.3",
     "org.hamcrest:hamcrest-library": "1.3",
     "org.javatuples:javatuples": "1.2",
-    "org.jetbrains.compose.animation:animation-core-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.desktop:desktop-jvm": "1.1.0-alpha04",
-    "org.jetbrains.compose.foundation:foundation-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.foundation:foundation-layout-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.material:material-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-geometry-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-graphics-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-test-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-test-junit4-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-text-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-unit-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.runtime:runtime-desktop": "1.1.0-alpha04",
-    "org.jetbrains.kotlin:kotlin-reflect": "1.6.0",
-    "org.jetbrains.kotlin:kotlin-stdlib-jdk7": "1.6.0",
-    "org.jetbrains.kotlin:kotlin-test": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-test": "1.6.0",
+    "org.jetbrains.compose.compiler:compiler": "1.3.0-alpha01",
```

Comment:
Note we import the compose compiler in a more traditional way now, rather than through a `http_file()` call.